Documentation/DataTypes/Properties/GetText.rst

```diff
@@ -147,7 +147,12 @@ DB
    Get the value of the header field of record with uid 234 from table
    "tt\_content"::
 
-      foo = DB : tt_content:234:header
+      foo.data = DB : tt_content:234:header
+      
+   Get the value of the header field of a record, whose uid is stored in a GET
+   parameter `myContentId`::
+   
+      foo.data.dataWrap = DB : tt_content:{GP : myContentId}:header
```

Comment:
We should mention that it is safe to directly use a client-/user-provided value here since internally the UID will be cast to integer and used as named parameter in the query.

This should raise awareness of SQL injections and hopefully avoid reports about a potential security issue here.

Reply:
Thanks, I added the information, this is important